.github/workflows/release.yml

```diff
@@ -756,7 +733,10 @@ jobs:
                   path: target/release
             - name: "Install and run upgrade test"
               run: |
-                  chmod uog+x target/release/*
+                  chmod uog+x target/release/tanssi-node
+                  chmod uog+x target/release/tanssi-relay
+                  chmod uog+x target/release/tanssi-relay-execute-worker
+                  chmod uog+x target/release/tanssi-relay-prepare-worker
```

Comment:
arent you missing  the container-chain binaries here?

Reply:
Ah, so that's the issue. Will just put `target/release/*` then